api/filter-alerts.go

```diff
@@ -0,0 +1,235 @@
+package api
+
+import (
+	"fmt"
+	graphql "github.com/cli/shurcooL-graphql"
+	"github.com/humio/cli/api/internal/humiographql"
+)
+
+type FilterAlert struct {
+	ID                 string   `graphql:"id"             yaml:"-"                       json:"id"`
+	Name               string   `graphql:"name"           yaml:"name"                    json:"name"`
+	Description        string   `graphql:"description"    yaml:"description,omitempty"   json:"description,omitempty"`
+	QueryString        string   `graphql:"queryString"    yaml:"queryString"             json:"queryString"`
+	Actions            []string `graphql:"actions"        yaml:"actions"                 json:"actions"`
+	Labels             []string `graphql:"labels"         yaml:"labels"                  json:"labels"`
+	Enabled            bool     `graphql:"enabled"        yaml:"enabled"                 json:"enabled"`
+	LastTriggered      int      `graphql:"lastTriggered"  yaml:"lastTriggered,omitempty" json:"lastTriggered,omitempty"`
+	LastErrorTime      int      `graphql:"lastErrorTime"  yaml:"lastErrorTime,omitempty" json:"lastErrorTime,omitempty"`
+	LastError          string   `graphql:"lastError"      yaml:"lastError,omitempty"     json:"lastError,omitempty"`
+	LastWarnings       []string `graphql:"lastWarnings"   yaml:"lastWarnings"            json:"lastWarnings"`
```

Comment:
Are these necessary? Since this API is meant for CRUD operations, I am not sure whether we should have them here.

Reply:
If the goal is CRUD operations, then yes, but for `filter-alert show` is seems like a nice to have?

```
...filter-alerts show DemoView some-filter-alert
                    ID | paer1zwIX3vsviZeQJ71Tuvlexxqtdym  
                  Name | some-filter-alert                 
               Enabled | false                             
           Description | some-description 2                
          Query String | some filter alert query 2         
                Labels | some-label-2                      
               Actions | testing                           
            Last Error |                                   
       Last Error Time | 0                                 
            Last Error |                                   
        Run As User ID | xxx          
  Query Ownership Type | User                              

Process finished with the exit code 0
```

Reply:
They just seem misleading when exporting, since you cannot import them again.

Reply:
I'll remove them

Reply:
```
                    ID | paer1zwIX3vsviZeQJ71Tuvlexxqtdym  
                  Name | some-filter-alert                 
               Enabled | false                             
           Description | some-description 2                
          Query String | some filter alert query 2         
                Labels | some-label-2                      
               Actions | testing                           
        Run As User ID | xxx          
  Query Ownership Type | User     
```